pkg/agent/apiserver/handlers/networkpolicy/handler.go

```diff
@@ -40,10 +40,11 @@ func HandleFunc(aq agentquerier.AgentQuerier) http.HandlerFunc {
 		npq := aq.GetNetworkPolicyInfoQuerier()
 		var nps []cpv1beta.NetworkPolicy
 
-		if npFilter.Pod != "" {
-			interfaces := aq.GetInterfaceStore().GetContainerInterfacesByPod(npFilter.Pod, npFilter.Namespace)
+		if pod != "" {
+			namespaceAndPodName := strings.Split(pod, "/")
```

Comment:
Here, are we sure "pod" must contain a single "/"? Or we should check that?

Reply:
Yes. `/` is not a valid character for Pod or Namespace name, and the string is generated by the `newFilterFromURLQuery` function only.

Reply:
Ok, so newFilterFromURLQuery already validated the API request inputs?

Reply:
Ah I see where you are going with this. Let me update it.

Reply:
On another thought, I'm not entirely sure that we should validate the `Pod` and `Namespace` input here contains only valid characters. If so, we should also add a validation for the other options in all other command, for example, the `-s` option which let you specify the ACNP/ANP/K8sNP name that the internal policy is derived from. If user provided a erroneous input, i.e. `antcl get networkpolicy -p test/pod -n test` (note the -n option is required when -p is specified), agent will not panic and will not give any result, which is the expected behavior, since the input is not valid. If you think we should add validation, then we probably would want to add validation for a lot more options.

Reply:
Humm.. my original question is more about if "pod" must have a least one `/`, so namespaceAndPodName will have at least two elements. Ideally, we should validate all resource names should not contain `/` (as otherwise the results can be unexpected, for example `-p p1 -n n1/p2`. But if we never validate this all for commands, we can fix it separately.

Reply:
To answer your 'original' question, yes it must have at least one `/` if it's not empty. See https://github.com/antrea-io/antrea/pull/3499/files#diff-d8d9be1f18febbeb4fa5e78eeb69b4e396d8b0867a3f0b4897f2474126676d01R91